### PR TITLE
[46] Product Page using HTML - Mobile Track

### PR DESCRIPTION
**Pull request by Oluwabusiolami**
## Task: `Name of given task`
- `Changes you made to given task`
Capture
Capture2
Capture3
Capture4

### CODE REVIEW

**Review by ainobs — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+         
+	  <li><a href="#">Cambridge Global English Workbook Stage 7</a></li>
+	  <h5>$23.30</h5>
+	  	  
+	  <li><a href="#">Cambridge Global English Workbook Stage 8</a></li>
+	  <h5>$23.30</h5>
+
+ 	  <li><a href="#">Cambridge Global English Teachers Resource Stage 7</a></li>
+	  <h5>$40.50</h5>
+
```
[the remark this answers is not in the document]
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:70]: Done

[the remark this answers is not in the document]
Comment (ainobs) [reply submissions/oluwabusolamiEluyela/html/index.html:70]: Nice one, keep it up

**Review by ainobs — APPROVED** [review]

**Review by ainobs — APPROVED** [review]

**Review by emmanuelidiaghe — APPROVED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:70]: What else am I supposed to do?

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+
+         <li><a href="#">New Releases</a></li><li> <a href="#">General Book Shelf</a></li><li> <a href="#">Today's Deals</a></li><li>
```
Comment (emmanuelidiaghe) [inline submissions/oluwabusolamiEluyela/html/index.html:18]: Hello Oluwabusiolami, you should begin each li tag on a new line; this makes your codes more readable.

**Review by emmanuelidiaghe — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html (hunk 2 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html]: Done. Please review

**Review by adaugochi — CHANGES_REQUESTED** [review]
ou are expected to make use of two images for this task. Remove other images that are not been used

**Comment by Oluwabusiolami** [comment]
I have just two images there. The break is as a result of the screenshot.

**Review by emmanuelidiaghe — APPROVED** [review]

**Comment by Oluwabusiolami** [comment]
What else am I to do to get this approved?

**Review by adaugochi — CHANGES_REQUESTED** [review]
You have about 6 images on this PR and you are only using 2, Remove the ones that are not used. 
image

**Comment by Oluwabusiolami** [comment]
> You have about 6 images on this PR and you are only using 2, Remove the ones that are not used.
> image

Done. I have removed them. Please review

**Comment by Oluwabusiolami** [comment]
I have effected the changes. Please review.

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    <header>
+      <div> 
+       <h1><a href="#">JUSTIFY</a></h1>
+      </div>
+	 <h2><a href="#">Hot Deals</a>
+         <a href="#">Gift Cards</a>
+         <a href="#">Sell</a><c/enter>
+         <a href="#">Book Rentals</a></h2>
+
+         <li><a href="#">New Releases</a></li>
```
Comment (adaugochi) [inline submissions/oluwabusolamiEluyela/html/index.html:18]: You are using `<li>` tag without `<ul>` or `<ol>` tag. This is wrong

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html (hunk 3 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:18]: Reviewed again. I have effected the changes. Please check.

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+         <li><a href="#">Registration</a></li> 
+	 <li><a href="#">Teachers Corner</a></li>
+	 <li><a href="#">Students Corner</a></li>
+         <li><a href="#">Contact</a></li>
+	 <li><a href="#">Order Offline</a></li>
+	</ol>
+  </header>
+   <main>
+    <div>
+     <li>
```
Comment (adaugochi) [inline submissions/oluwabusolamiEluyela/html/index.html:30]: You are using `<li>` tag without `<ul>` or `<ol>` tag. This is wrong. And it isn't used properly in this context

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+             Written by well-respected authors, the Cambridge Checkpoint Mathematics suite provides a comprehensive structured resource 
+	     which covers the full Cambridge Secondary 1 Mathematics framework in three stages. This brightly illustrated Coursebook for 
+	     Stage 7 offers a comprehensive introduction to all topics covered in the syllabus. Worked examples show students how to tackle 
+	     different problems, and plenty of exercise questions prepare students for the different types of question they will face in their 
+	     exam. Coverage of the Problem Solving framework is integrated throughout the course, with questions relating to the framework 
+	     statements highlighted in the Coursebook and Practice Book.
+            </div>
+          </li>
+          <hr />
+          <li>
```
Comment (adaugochi) [inline submissions/oluwabusolamiEluyela/html/index.html:48]: You are using `<li>` tag without `<ul>` or `<ol>` tag. This is wrong. And it isn't used properly in this context

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html (hunk 4 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html:31]: Reviewed. Please check

Comment (adaugochi) [reply submissions/oluwabusolamiEluyela/html/index.html:31]: Welldone @Oluwabusiolami But I don't think ol and li tag is useful in this scenario or usage. I will advise that you remove them

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+	     different problems, and plenty of exercise questions prepare students for the different types of question they will face in their 
+	     exam. Coverage of the Problem Solving framework is integrated throughout the course, with questions relating to the framework 
+	     statements highlighted in the Coursebook and Practice Book.
+            </div>
+          </li>
+	</ol>
+          <hr/>
+	<ol>
+          <li>
+            <div>
```
Comment (ToluAkin) [inline submissions/oluwabusolamiEluyela/html/index.html:52]: @Oluwabusiolami You did not close this `div`

File: submissions/oluwabusolamiEluyela/html/index.html [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+	 <li><a href="#">Today's Deals</a></li>
+         <li><a href="#">Registration</a></li> 
+	 <li><a href="#">Teachers Corner</a></li>
+	 <li><a href="#">Students Corner</a></li>
+         <li><a href="#">Contact</a></li>
+	 <li><a href="#">Order Offline</a></li>
+      </ol>
+  </header>
+   <main>
+    <div>
```
Comment (ToluAkin) [inline submissions/oluwabusolamiEluyela/html/index.html:29]: @Oluwabusiolami You did not close this `div`

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/oluwabusolamiEluyela/html/index.html (hunk 7 above)
Comment (Oluwabusiolami) [reply submissions/oluwabusolamiEluyela/html/index.html]: It is well. I have removed it now. I hope it will be approved now @adaugochi @ToluAkin

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**